iree/compiler/Dialect/HAL/Transforms/BUILD

```diff
@@ -43,14 +43,19 @@ cc_library(
         "//iree/compiler/Dialect/HAL/Target",
         "//iree/compiler/Dialect/IREE/IR",
         "//iree/compiler/Dialect/Shape/Transforms",
-        "//iree/compiler/Utils",
+        "//iree/compiler/Dialect/VM/IR",
         "@com_google_absl//absl/strings",
         "@llvm-project//llvm:Support",
+        "@llvm-project//mlir:Affine",
+        "@llvm-project//mlir:GPUDialect",
         "@llvm-project//mlir:IR",
+        "@llvm-project//mlir:LinalgOps",
         "@llvm-project//mlir:Pass",
+        "@llvm-project//mlir:SCFDialect",
         "@llvm-project//mlir:StandardOps",
         "@llvm-project//mlir:Support",
         "@llvm-project//mlir:Transforms",
+        "@llvm-project//mlir:VectorOps",
```

Comment:
HAL transforms do not depend on these and we shouldn't be pulling them in.

Reply:
Yeah leftover from debugging. Thanks for catching.